build/Helix/packages.config

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="TAEF.Redist.Wlk" version="10.57.200918006-develop" targetFramework="native" />
+  <package id="Microsoft.Taef.Redist.Wlk" version="10.58.210126003-develop" targetFramework="native" />
```

Comment:
>Microsoft [](start = 15, length = 9)

is switching to reversed namespace the recommended fix?

Reply:
Reversed namespace? Not sure if that's what you mean by having Microsoft in the front. Yup this was recommended to me. All I did was edit the nuspec in the nukpg package I uploaded so the changes are local.

Reply:
Nope but I am asking the TAEF team to rename or sign their package. Since I it's only the tests that are using this and we don't update it that frequently, I thought this should be okay as a option.

Reply:
This is a public feed. I am not sure whether we need to wait for "official" feed.

Reply:
Yea we won't but it might be a good idea to do so now that we have these issues.